code/game/mecha/working/hamsandwich.dm

```diff
@@ -0,0 +1,42 @@
+/obj/mecha/working/hamsandwich
+	desc = "Combining man and machine for a better, stronger engineer."
+	name = "Hamsandwich"
+	icon_state = "hamsandwich"
+	initial_icon = "hamsandwich"
+	step_in = 1
+	step_energy_drain = 2
+	max_temperature = 100000
+	health = 100
+	wreckage = /obj/effect/decal/mecha_wreckage/ripley
+	max_equip = 4
+	var/image/thruster_overlay
+
+/obj/mecha/working/hamsandwich/New()
+	..()
+	thruster_overlay = image('icons/mecha/mecha.dmi', src, "[initial(icon_state)]-thruster_overlay")
+
+/obj/mecha/working/hamsandwich/check_for_support()
+	return 1
+
+/obj/mecha/working/hamsandwich/mechturn(direction)
```

Comment:
This proc is exactly the same as the overridden.

Reply:
No, it uses a different .ogg file than `/obj/mecha/mechturn()` does.

Reply:
Ah, whoops.

Though it might be a good idea to not have to duplicate the dir changing code.